packages.json

```
@@ -1,4 +1,9 @@
 [
+    
+    {
+        "package": "RJDemetra",
+        "url": "https://github.com/rjdverse/rjdemetra"
+    },
```

Do you want to publish the dev version or the latest release on r-universe?
```suggestion
        	"branch" : "*release"
    },
```

---

Both are justified, but all the other packages have been released on r-universe.

---

No only the dev version to facilitate it's installation, the release version is already on CRAN so easily installable.